packages/cli/src/PublicApi/v1/handlers/users/users.service.ee.ts

```diff
@@ -25,9 +27,12 @@ export async function getAllUsersAndCount(data: {
 	includeRole?: boolean;
 	limit?: number;
 	offset?: number;
+	in?: string[];
 }): Promise<[User[], number]> {
+	const { in: _in } = data;
+
 	const users = await Container.get(UserRepository).find({
-		where: {},
+		where: { ...(_in && { id: In(_in) }) },
 		skip: data.offset,
 		take: data.limit,
 	});
```

Comment:
eventually we should move all pagination code like this into a base repository, and make every "get-many" endpoint use the same code. we have more versions of pagination code than I'd like us to have.